src/Moq/Linq/Mock.cs

```diff
@@ -45,6 +56,29 @@ public partial class Mock
 			return mock.Object;
 		}
 
+		/// <summary>
+		/// Creates a mock object of the indicated type.
+		/// </summary>
+		/// <param name="behavior">Behavior of the mock.</param>
+		/// <param name="args">Optional constructor arguments if the mocked type is a class.</param>
+		/// <typeparam name="T">The type of the mocked object.</typeparam>
+		/// <returns>The mocked object created.</returns>
+		public static T Of<T>(MockBehavior behavior,params object[] args) where T : class
+		{
+			// This method was originally implemented as follows:
```

Comment:
This is copied from the overload above but I doubt if this historical improvement-insight should have his place in code. I would leave this at least in this copy imho

Reply:
Agreed, that comment is a bit out of place since the method it appears in didn't even exist at the time that the comment refers to.

It's a detail and not super important, but if you're going to edit anyway consider removing this comment.

Reply:
You are right @peterdew and @stakx. When I copied the original method I forgot to remove the comment, sorry. I will remove it